amqp_influxdb/__main__.py

```diff
@@ -20,13 +20,21 @@
     InfluxDBPublisher,
     AMQPTopicConsumer,
     BATCH_SIZE,
-    MAX_BATCH_DELAY)
+    MAX_BATCH_DELAY,
+    BROKER_PORT_SSL,
+    BROKER_PORT_NO_SSL,
+)
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--amqp-hostname', required=False,
                         default='localhost')
+    parser.add_argument('--amqp-username', required=True)
```

Comment:
This was done after discussion with @nir0s and @noakup - the decision was taken that the username and password should be required across cloudify for using the broker/